.buildkite/test-template.j2

```diff
@@ -0,0 +1,24 @@
+{% set docker_image = "public.ecr.aws/q9t5s3a7/vllm-ci-test-repo:$BUILDKITE_COMMIT" %}
+{% set default_working_dir = "/vllm-workspace/tests" %}
+{% set hf_home = "~/.cache/huggingface" %}
+
+steps:
+  {% for step in steps %}
+  {% if step.num_nodes >= 2 %}
+  - label: Multinode test
+    priority: 1000
+    agents:
+      queue: gpu_4_queue
+    commands:
+      - docker network create --subnet=192.168.10.0/24 docker-net
```

Comment:
what happens if the test breaks, will the network be deleted? if not, will this network creation fail on subsequent test?